src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php

```diff
@@ -153,7 +153,14 @@ public function start()
         }
 
         // ok to try and start the session
```

Comment:
this comment should be moved, shouldn't it?

Reply:
I don't have a strong feeling about this but I think it does belong where it is, in so far as this means, "we will now proceed to start the session since we have reached this code path." Since the code that follows is directly related to starting the session, I think this belongs before that code, but again, no strong feelings here other than maybe reducing the size of the diff.